cluster/ci/config/prow/plugins.yaml

```diff
@@ -448,7 +448,7 @@ config_updater:
       name: prow-job-cluster-launch-e2e-openshift-ansible
     ci-operator/templates/cluster-launch-e2e.yaml:
       name: prow-job-cluster-launch-e2e
-    ci-operator/templates/cluster-launch-installer-e2e.yaml:
+    ci-operator/templates/openshift/master/cluster-launch-installer-e2e.yaml:
```

Comment:
This will need a manual upload to bootstrap

Reply:
Or this change needs to merge before the files are actually changed (so the updater can have up-to-date config when the files merge

Reply:
> Or this change needs to merge before the files are actually changed...

Done via #1768.